src/renderer/containers/Console/index.tsx

```diff
@@ -74,6 +74,15 @@ export const Console: React.FC = () => {
     // Close the dialog
     onCancel();
   };
+
+  ipc_consoleMirrorErrorMessageEvent.renderer!.handle(async ({ message }) => {
```

Comment:
This should be `useEvent` instead of `handle` since `handle` expects the renderer to clean up and unsubscribe.

Also you should consider when we want to unsubscribe. If you add this line of code inside of the `Console` component, if you happen to have browsed to a different page, it would have unsubscribed already so you miss the messages. Global events such as these which should be subscribed to regardless of current page should probably be moved to `useAppListeners`.